runtime/wasm/runtime.js

```diff
@@ -143,6 +143,18 @@
     return h ^ s.length;
   }
 
+  function getenv(n) {
+    if (isNode && process.env[n] !== undefined) return process.env[n];
+    return globalThis.jsoo_env?.[n];
+  }
+
+  let record_backtrace_flag = 0;
+
+  for (const l of getenv("OCAMLRUNPARAM")?.split(",") || []) {
+    if (l === "b") record_backtrace_flag = 1;
+    if (l.startsWith("b=")) record_backtrace_flag = +!!+l[i].slice(2);
```

Comment:
Why do we need the additional `+!!` ?

Reply:
To make sure that we value of the flag is a boolean (0 or 1), not an arbitrary number.

Reply:
I think I would prefer a more explicit
`+l[i].slice(2) ? 1 : 0`